server/models/Profile.js

```diff
@@ -19,11 +19,6 @@ const profileSchema = new mongoose.Schema({
     required: true,
     default: false,
   },
-  isSitter: {
-    type: Boolean,
-    required: true,
-    default: false,
-  },
```

Comment:
Is this because we are now using mongoose discriminators? If not, can you explain why we are getting rid of this?

And just as a quick note, this is the sort of information that would be best placed in the PR description 👍🏼

Reply:
For some reason lines 22-26 were duplicate of the 5 lines above so I was fixing that as well. Thanks for the helpful PR feedback

Reply:
Yeah I missed this, thanks 👍🏼